Code/Tools/ProjectManager/Source/ProjectExportWorker.cpp

```diff
@@ -67,6 +68,15 @@ namespace O3DE::ProjectManager
         return AZ::Success(logFilePath.filePath(ProjectExportErrorLogName));
     }
 
+    AZ::Outcome<QString> ProjectExportWorker::GetExpectedOutputPath() const
+    {
+        if (!m_expectedOutputDir.isEmpty())
+        {
+            return AZ::Success(m_expectedOutputDir);
+        }
+        return AZ::Failure();
+    }
+
```

Comment:
There seems to be 2 reasons for failure of getting the output path. 
1. The output path was not captured from the log
2. The output path is not a valid (or absolute) directory. 
Lets change the return type to `AZ::Outcome<QString, QString>`

```suggestion
    AZ::Outcome<QString, QString> ProjectExportWorker::GetExpectedOutputPath() const
    {
        if (m_expectedOutputDir.isEmpty())
        {
            return AZ::Failure(_tr("Project Export Output folder not detected in the output logs"));
        }
        if (QDir(m_expectedOutputDir).isAbsolute())
        {
            return AZ::Failure(_tr("Project Export Output folder %s is invalid.").arg(m_expectedOutputDir));
        }
        return AZ::Success(m_expectedOutputDir);
    }

```